packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -68,6 +68,8 @@ export const messages = {
 
   force_org_create_default_scratch_org_text:
     'SFDX: Create a Default Scratch Org...',
+  force_org_create_result_parsing_error:
```

Comment:
@hbc-bindu This is the only new message that we'll potentially surface to users.

Reply:
@lcampos, you could rephrase the message as:
An unexpected error occurred while processing the org create response.
PR seems to have been merged into develop; please update, if possible.